.gitignore

```diff
@@ -106,3 +106,9 @@ properties/jacoco/**
 Deploy/website-temp
 ### Snyk CLI
 .dccache
+# Ignore the unencrypted secret
+secrets/splunk_oncall_email_prod.sops.yml
```

Comment:
We shouldn't need to have an ignore specific to this one secret. Instead of adding a new encrypted file for this secret, I think it should be added as an "/ab2d/mgmt/" value to the files in ops/services/00-bootstrap/values.